lib/VerifiableCredentialStore.js

```diff
@@ -126,12 +162,16 @@ export default class VerifiableCredentialStore {
         invocationSigner
       });
       if(!doc) {
-        return false;
+        const err = new Error('Verifiable Credential not found.');
+        err.name = 'NotFoundError';
+        throw err;
```

Comment:
I have no idea if this is what is desired or not -- noting it's a breaking change though.